### PR TITLE
Updated some files for better GUI

### PR DESCRIPTION
**Pull request by ShahzadUmarBaig**
Check the GUI please #40

### CODE REVIEW

File: lib/screens/activity_edit_sheet.dart [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import 'package:flutter/cupertino.dart' as c;
 import 'package:flutter/widgets.dart';
+import 'package:star_book/models/mood.dart';
+import 'package:star_book/utils/color.dart';
+import 'package:star_book/widgets/color_container.dart';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:5]: Use relative import.
Add them below //Files

File: lib/screens/activity_edit_sheet.dart [hunk 2]
```diff
@@ -23,6 +26,7 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
   Activity activity;
   final TextEditingController titleController = TextEditingController();
   final TextEditingController noteController = TextEditingController();
+  var containerColor;
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:29]: I think that there is no need for this variable

File: lib/screens/activity_edit_sheet.dart [hunk 3]
```diff
@@ -93,13 +105,15 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
               onTap: null,
             ),
             Padding(padding: EdgeInsets.symmetric(vertical: 8)),
-            ActionContainer(
+            ColorContainer(
               text: "Mood",
-              icon: c.CupertinoIcons.right_chevron,
+              color: containerColor,
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:110]: Make a function to get container color

File: lib/screens/activity_edit_sheet.dart [hunk 4]
```diff
@@ -70,6 +74,14 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
   }
 
   Widget _buildBody(BuildContext context) {
+    // Before we return the body of the main context/activity sheet, it checks the conditions everytime
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:77]: return the result from that function based on the condition.

File: lib/widgets/color_container.dart [hunk 5]
```diff
@@ -29,14 +29,13 @@ class ColorContainer extends StatelessWidget {
                 c.CupertinoColors.label,
                 context,
               ),
-              fontSize: 20,
             ),
           ),
           Container(
-            padding: EdgeInsets.symmetric(vertical: 12, horizontal: 12),
+            padding: EdgeInsets.symmetric(vertical: 8, horizontal: 8),
```
Comment (hashirshoaeb) [inline lib/widgets/color_container.dart:35]: 8 seems a bit small. what do you say?

Comment (ShahzadUmarBaig) [reply lib/widgets/color_container.dart:35]: 8 Looks better but Let me change it to 10

File: lib/screens/activity_edit_sheet.dart (hunk 4 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:77]: Yes let me fix that using the method.

File: lib/screens/activity_edit_sheet.dart (hunk 3 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:110]: Working on this

File: lib/screens/activity_edit_sheet.dart (hunk 1 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:5]: Okay let me check what do you mean.

**Comment by ShahzadUmarBaig** [comment]
Completed all of the tasks, you can review them and let me know for any further changings

File: lib/screens/activity_edit_sheet.dart [hunk 6]
```diff
@@ -2,11 +2,12 @@ import 'package:flutter/cupertino.dart' as c;
 import 'package:flutter/widgets.dart';
 // Files
 import '../routes/route_generator.dart';
-import '../widgets/action_container.dart';
 import '../models/activity.dart';
 import '../widgets/my_container.dart';
 import '../controllers/activity.dart';
 import '../styles/style.dart';
+import 'package:star_book/utils/color.dart';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:9]:
```suggestion
import '../utils/color.dart';
```

File: lib/screens/activity_edit_sheet.dart [hunk 7]
```diff
@@ -2,11 +2,12 @@ import 'package:flutter/cupertino.dart' as c;
...
+import 'package:star_book/widgets/color_container.dart';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:10]:
```suggestion
import '../widgets/color_container.dart';
```

File: lib/screens/activity_edit_sheet.dart [hunk 8]
```diff
@@ -54,6 +55,14 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
     Navigator.of(context, rootNavigator: true).pop();
   }
 
+  c.CupertinoDynamicColor _getMoodColor() {
+    return activity.moodId == null
+        ? c.CupertinoColors.systemGrey
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:60]:
```suggestion
        ? c.CupertinoDynamicColor.resolve( c.CupertinoColors.tertiarySystemGroupedBackground, context)
```

File: lib/screens/activity_edit_sheet.dart [hunk 9]
```diff
@@ -54,6 +55,14 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
...
+        : getColor(
+            EColor.values[activity.moodId],
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:62]:
```suggestion
            c.CupertinoDynamicColor.resolve(EColor.values[activity.moodId], context)
```

File: lib/screens/activity_edit_sheet.dart (hunk 8 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:60]: No point doing this.

File: lib/screens/activity_edit_sheet.dart (hunk 9 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:62]: Screenshot (111)

ref

File: lib/screens/activity_edit_sheet.dart (hunk 8 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:60]: fixed

File: lib/screens/activity_edit_sheet.dart (hunk 7 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:10]: fixed

File: lib/screens/activity_edit_sheet.dart (hunk 6 above)
Comment (ShahzadUmarBaig) [reply lib/screens/activity_edit_sheet.dart:9]: fixed

File: lib/screens/activity_edit_sheet.dart (hunk 9 above)
Comment (hashirshoaeb) [reply lib/screens/activity_edit_sheet.dart:62]:
```dart
  c.CupertinoDynamicColor _getMoodColor() {
    return c.CupertinoDynamicColor.resolve(
      activity.moodId == null
          ? c.CupertinoColors.tertiarySystemGroupedBackground
          : getColor(
              EColor.values[activity.moodId],
            ),
      context,
    );
```

**Pull request merged**